sklearn/inspection/tests/test_permutation_importance.py

```diff
@@ -485,3 +526,30 @@ def test_permutation_importance_multi_metric(list_single_scorer, multi_scorer):
         )
 
         assert_allclose(multi_result.importances, single_result.importances)
+
+
+def test_error_max_samples():
+    pd = pytest.importorskip("pandas")
```

Comment:
I assume that we don't need to have pandas. We could use a NumPy array and thus run the test all the time.

Reply:
Agreed, the test was overly complicated